src/controllers/login-controller.js

```diff
@@ -1,33 +1,58 @@
 const Accounts = require('../models/accounts');
-const {sendToTelegram} = require('../helpers/telegramBot');
-const  {tokenSing}= require('../helpers/generateToken');
-const {jsonErrors} = require('../helpers/exception-responses');
-const {ResponseREST} = require('../helpers/responses');
-const {Number} = require('../helpers/number');
-
+const {tokenSing}= require('../helpers/generateToken');
+const {StatusCodes} = require ('http-status-codes');
+const {responseServer, responseGeneral} = require('../helpers/response-result');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
const {responseServer, responseGeneral} = require("../helpers/response-result");
```